fideslib/core/config.py

```diff
@@ -194,9 +193,7 @@ def load_file(file_names: Union[List[Path], List[str]]) -> str:
         for file_name in file_names:
             possible_location = os.path.join(dir_str, file_name)
             if possible_location and os.path.isfile(possible_location):
-                logger.info(
-                    "Loading file %s from %s", NotPii(file_name), NotPii(dir_str)
-                )
+                logger.info("Loading file %s from %s", file_name, dir_str)
```

Comment:
I debated this. I ended up removing it because the only place it was used was in the config which will generally be overridden. Also, I didn't see it being used in fideslib so it seems to be fidesops specific and therefore didn't fit fideslib.

That said if you think it should be here it is easy to add back.